docs/content/dagster-cloud/developing-testing/code-locations.mdx

```diff
@@ -1,7 +1,245 @@
 ---
-title: "Adding code locations to Dagster Cloud  | Dagster Docs"
+title: "Managing code locations in Dagster Cloud | Dagster Docs"
 
 noindex: True
 ---
 
-# Adding code locations to Dagster Cloud
+# Managing code locations in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll cover the requirements for Dagster code, how to add code in Dagster Cloud, and how to add code using the `dagster-cloud` CLI.
+
+---
+
+## Understanding code locations
+
+<Note>
+  Learn by example? Check out the{" "}
+  <a href="https://github.com/dagster-io/hooli-data-eng-pipelines">
+    example repo
+  </a>
+  , which is set up to run in Dagster Cloud.
+</Note>
+
+For an example of a Github repo set up to run in Dagster Cloud, see our
+
+A code location specifies a single Python package or file that defines your Dagster code. When you add a code location in Dagster Code, you're instructing the agent where to find your code.
+
+When you add or update a code location, the agent uses the location configuration to load your code and upload metadata about your jobs to Dagster Cloud. Each Dagster Cloud instance deployment can include code from one or more code locations.
+
+Note that, unlike Dagster Open Source, Dagster Cloud doesn't require a `workspace.yaml` file. Instead, you use the Dagster Cloud API to configure your workspace. You can still create a `workspace.yaml` file if you want to load your code in an open-source Dagit instance, but doing so won't affect how your code is loaded in Dagster Cloud.
+
+---
+
+## Dagster Cloud code requirements
+
+To work with Dagster Cloud, your Dagster code:
+
+- **Must be loaded from a single entry point, either a Python file or package.** This entry point can load repositories from other files or packages.
+
+- **Must run in an environment where the `dagster` and `dagster-cloud` 0.13.2 or later Python packages are installed.**
```

Comment:
maybe "the `dagster` and `dagster-cloud` packages at version 0.13.2 or later are installed"? something about "where the `dagster` and `dagster-cloud` 0.13.2 or later Python packages" isn't parsing for me

Reply:
Yeah, reading this again I agree it's awkward. I like your phrasing better.